spec/factories/user.rb

```diff
@@ -1,22 +1,22 @@
 # This will guess the User class
 FactoryGirl.define do
   factory :user do
-    email                  "user@example.com"
-    password               "password"
-    password_confirmation  "password"
+    email 'user@example.com'
+    password 'password'
+    password_confirmation 'password'
   end
 
   factory :curator, class: User do
-    email                  "curator@example.com"
-    password               "password"
-    password_confirmation  "password"
-    after(:build)  { |user| user.webauth_groups = "dlss:exhibits-creators" }
+    email 'curator@example.com'
+    password 'password'
+    password_confirmation 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.